ChangeLog

```diff
@@ -1,3 +1,29 @@
+Wed Apr 20 2022 Weronika Lewandowska <weronika.lewandowska@intel.com>
+
+	* Version 1.12.0-rc1
+
+	This release:
+	- Deprecate librpmem library and rpmemd tool.
+	Using the librpmem library API will result in warnings
+	and is no longer recommended. Those interested in a remote
+	persistent memory support should use the new rpma library.
+	- Introduces a new API in libpmem2 to perform asynchronous data
+	movement operations. This feature requires miniasync(7) library
+	dependention.
```

Comment:
To use this feature, software needs to include an optional miniasync dependency.